test/Dialect/Calyx/errors.mlir

```diff
@@ -708,3 +741,16 @@ calyx.program {
     calyx.control { calyx.enable @A }
   }
 }
+
+// -----
+
+calyx.program {
+
+  calyx.component @main(%go: i1 {go}, %clk: i1 {clk}, %reset: i1 {reset}) -> (%done: i1 {done}) {
+    // expected-error @+1 {{'calyx.std_gt' op with static value: 1. Cannot have non-zero static value and Combinational trait.}}
+    %gt.left, %gt.right, %gt.out = calyx.std_gt "gt" {static=1} : i8, i8, i1
```

Comment:
Cell definitions don’t usually have a static attribute on them. It’s usually the primitive itself that might define a static attribute. Calyx has an explicit comb keyword to define primitives that are combinational (see the core.futil)

Reply:
Good point. I was thinking maybe that some definitions may be determined to be combinational on the fly, but even for instances, it would be an instance of a combinational component.

Reply:
I'm going to leave an assertion for now, just so that someone doesn't throw the combinational trait on anything that takes > 0 cycles.